.github/workflows/php.yml

```diff
@@ -0,0 +1,61 @@
+name: PHP Composer
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: ['7.2', '7.3', '7.4']
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ matrix.php-version }}
+
+    - name: Validate composer.json and composer.lock
```

Comment:
Does the “and composer.lock” part do anything? We don’t commit the `composer.lock` part in this repository, as far as I can tell.

Reply:
Nope, it doesn't do anything as the files doesnt exit.
We could remove it from the name, it was only included as this is the standard github actions suggested name